scripts/awsicons/icon.py

```diff
@@ -68,6 +68,14 @@ def generate_puml(self, path):
                 target=self.target, color=self.color
             )
 
```

Comment:
For this section, could you remove the blank lines between `puml_content +=` as with the entity lines?

Reply:
Ok, any reason not to use f'' format strings?

Reply:
Yeah, I need to refactor all to f'' strings from .format(). I'd started with making the builder 2.x and 3.x compatible, but will 3.x compatible so will open an issue to refactor.